admin_guide/docker_registry.adoc

```diff
@@ -87,79 +99,101 @@ $ oc exec -p docker-registry-1-da73t find /registry
 /registry/docker/registry/v2/repositories/p1/pause/_layers/sha256/f72a00a23f01987b42cb26f259582bb33502bdb0fcf5011e03c60577c4284845
 /registry/docker/registry/v2/repositories/p1/pause/_layers/sha256/f72a00a23f01987b42cb26f259582bb33502bdb0fcf5011e03c60577c4284845/link
 ----
-<1> This directory stores all layers and signatures as blobs
-<2> This file contains the blob's contents
-<3> This directory stores all the image repositories
-<4> This directory is for a single image repository `p1/pause`
-<5> This directory contains signatures for a particular image manifest revision
-<6> This file contains a reference back to a blob (which contains the signature data)
-<7> This directory contains any layers that are currently being uploaded and staged for the given repository
-<8> This directory contains links to all the layers this repository references
-<9> This file contains a reference to a specific layer that has been linked into this repository via an image
+<1> This directory stores all layers and signatures as blobs.
+<2> This file contains the blob's contents.
+<3> This directory stores all the image repositories.
+<4> This directory is for a single image repository *p1/pause*.
+<5> This directory contains signatures for a particular image manifest revision.
+<6> This file contains a reference back to a blob (which contains the signature
+data).
+<7> This directory contains any layers that are currently being uploaded and
+staged for the given repository.
+<8> This directory contains links to all the layers this repository references.
+<9> This file contains a reference to a specific layer that has been linked into
+this repository via an image.
 
 == Securing the Registry
 
-=== Overview
-
-You can optionally secure the registry to serve traffic via TLS.
-
-=== Procedure
+You can optionally secure the registry to serve traffic via TLS:
```

Comment:
Verb part of this sentence sounds awkward.  Maybe "optionally arrange for the registry to"?